pylintrc

```diff
@@ -4,6 +4,7 @@ disable=
     C0115,
     C0116,
     R0903,
+    E1123
```

Comment:
Why are we disabling the `unexpected-keyword-arg` rule globally?

Reply:
Because pylinter is a bitch and is not working properly.